### PR TITLE
Update stalebot message to close issues

### PR DESCRIPTION
**Pull request by pedroalvesbatista**
## Description of the Pull Request (PR):

Updates informations, such as message and interval for stalebot to close issues.

#### Before submitting a PR, make sure you have done the following:

- Read the [Guidelines for Contributing](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/hpcng/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR, linted with `make check`  and tested this PR locally with a `make test`, and `make testall` if possible (see CONTRIBUTING.md).
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/hpcng/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/hpcng/singularity/blob/master/CONTRIBUTORS.md)

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -26,21 +26,31 @@ exemptMilestones: false
 exemptAssignees: false
 
 # Label to use when marking as stale
-staleLabel: stale
+staleLabel: close
```
Comment (DrDaveD) [inline .github/stale.yml:29]: I suggest `autoclose` instead of `close`.

File: .github/stale.yml [hunk 2]
```diff
@@ -26,21 +26,31 @@ exemptMilestones: false
...
-  This issue has been automatically marked as stale because it has not had
-  activity in over 180 days. It will be closed in 14 days if no further activity occurs. Thank you
-  for your contributions.
+  Dear user/contributor,
+
+Since the Singularity project has joined the Linux Foundation and been renamed
+to Apptainer, we are closing issues in this repository.  If this issue is still 
+important to you, please open a new [Apptainer issue](https://github.com/apptainer/apptainer/issues) that links back to this one.
+
+We appreciate your contribution and feedback.
```
Comment (DrDaveD) [inline .github/stale.yml:39]: It doesn't make sense to have this comment repeated.  Set this to `false`.

File: .github/stale.yml [hunk 3]
```diff
@@ -26,21 +26,31 @@ exemptMilestones: false
...
 
 # Comment to post when removing the stale label.
 # unmarkComment: >
 #   Your comment here.
 
 # Comment to post when closing a stale Issue or Pull Request.
 closeComment: >
-   This issue has been automatically closed because no response was provided within 14 days.
+   TDear user/contributor,
```
Comment (DrDaveD) [inline .github/stale.yml:47]: Typo, remove the leading `T` (and the leading blanks).

**Review by DrDaveD — CHANGES_REQUESTED** [review]
@kmuriki @pedroalvesbatista 

Have you gone through the issues to select the obvious ones to keep, created new issues in apptainer for them, and closed the corresponding singularity issues?  That has to be done before this is merged.

File: .github/stale.yml (hunk 1 above)
Comment (pedroalvesbatista) [reply .github/stale.yml:29]: Got it !

**Comment by DrDaveD** [comment]
The plan is for @kmuriki to instead configure [repo-lockdown](https://github.com/dessant/repo-lockdown).  Closing this.

**Pull request closed without merging**